### DISCUSSION TITLE
Any way to make make the app work on Wayland?

### DISCUSSION THREAD

**Discussion by mahdihasan2006** [ActivityWatch/activitywatch#1109] [Q&A] [answered]
The program worked on X11 session. But on Wayland. Is there any way to make it work on Wayland session?
By the way, I am using Fedora 41 and ActivityWatch v0.13.2

**Comment by ErikBjare** [comment, marked as answer]
It depends, but I think the best current workaround is to use `awatcher` which supports several Wayland compositors: https://github.com/ActivityWatch/activitywatch/issues/92#issuecomment-1577843623

**Reply by abdelkareemkobo** [reply]
@ErikBjare Hi, Erik. 
thanks for creating such a great tool. 
i search if it's supports wayland but there is multiple answers and they are old. 
what is the current status now?
i wan to use it on Cosmic Os from popos in their wayland!
